.github/workflows/build-and-test.yaml

```diff
@@ -75,13 +94,31 @@ jobs:
         echo "effective_version=${effective_version}" >> "${GITHUB_OUTPUT}"
         echo "repo_version=${src_version}" >> "${GITHUB_OUTPUT}"
         echo "setuptools_version=${setuptools_version}" >> "${GITHUB_OUTPUT}"
+    - name: create commit with effective version
+      if: ${{ inputs.release }}
+      run: |
+        git config --global --add safe.directory $PWD
+        git config user.name 'Gardener-CICD Bot'
+        git config user.email 'gardener.ci.user@gmail.com'
+        echo ${{ steps.version.outputs.effective_version }} | .ci/write-version
+        touch /tmp/timestamp-ref
```

Comment:
Don't we have to guard against cases where `/tmp` is not part of the `.gitignore` file?

Reply:
by default, `/tmp` is outside of repository-tree, so: no :-)

However, I think we should anyhow maybe switch to a mechanism that does not need a reference-file. For example, we might consider to explicitly set old timestamps (as in epoch-0) after git-clone. that way, any file from worktree might be used as reference-file, thus making the interface less clumsy